package.json

```diff
@@ -17,7 +17,7 @@
     "url": "http://github.com/nock/nock/issues"
   },
   "engines": {
-    "node": ">= 8.0"
+    "node": ">= 10.0"
```

Comment:
If the idea is to only support LTS, this should be `>=10.13`. 
https://nodejs.org/en/download/releases/

Reply:
I think LTS and Active.

Reply:
We should be supporting LTS Active and Maintenance.
https://nodejs.org/en/about/releases/
Otherwise we'd be dropping 10.x in two months.

Reply:
I think the ones we support on that chart are Current, Active, and Maintenance. Does that sound right?